frontend/packages/kubevirt-plugin/src/components/vm-templates/vm-template-resource.tsx

```diff
@@ -15,23 +14,48 @@ import {
 import { ResourceSummary } from '@console/internal/components/utils';
 import { DASH } from '@console/shared';
 import { TemplateKind, K8sResourceKind } from '@console/internal/module/k8s';
+
 import { VMTemplateLink } from './vm-template-link';
+import { getId, prefixedId } from '../../utils';
+import { vmDescriptioneModal } from '../modals/vm-description-modal';
+import { getDescription } from '../../selectors/selectors';
+
+import '../vms/_vm-resource.scss';
```

Comment:
Why not have a dedicated stylesheet & do the import there?

Using the same stylesheet for both VM & VM template is dangerous, since they are logically two separate objects that might diverge over time in terms of logic and styling.

Reply:
Made the stylesheets less dependent. Not sure about the location of `_resource-summary-description.scss` though